Fix for CNV-44887 (#83477)
openshift/openshift-docs: virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc

link:https://docs.openshift.com/pipelines/latest/about/understanding-openshift-pipelines.html[{pipelines-title}] is a Kubernetes-native CI/CD framework that allows developers to design and run each step of the CI/CD pipeline in its own container.

> The {pipelines-shortname} tasks and example pipeline allow you to do the following:

Reviewer: ```suggestion
By using {pipelines-shortname} tasks and the example pipeline, you can complete the following tasks:
```

Reviewer: "allow you" is a form of anthrophomorphism.

> * Create and manage virtual machines (VMs), persistent volume claims (PVCs), data volumes and data sources

Reviewer: I think periods are needed here as these are complete sentences.

> * Create and manage virtual machines (VMs), persistent volume claims (PVCs), data volumes and data sources

Reviewer: ```suggestion
* Create and manage virtual machines (VMs), persistent volume claims (PVCs), data volumes, and data sources
```

* Run commands in VMs
* Manipulate disk images with `libguestfs` tools

The tasks are located in link:https://artifacthub.io/packages/search?repo=redhat-tekton-tasks&sort=relevance&page=1[the task catalog].

> The example Windows pipeline is located in link:https://artifacthub.io/packages/tekton-pipeline/redhat-pipelines/windows-efi-installer[the pipeline catalog].

Reviewer: Link points to an article called "Windows EFI Installer Pipeline". Consider adding (ArtifactHub) after the article name so customers know they need to leave the OCP docs.
